content/en/docs/setup/install/standalone-operator/index.md

```diff
@@ -171,9 +171,59 @@ $ kubectl logs -f -n istio-operator $(kubectl get pods -n istio-operator -lname=
 Refer to the [`IstioOperator` API](/docs/reference/config/istio.operator.v1alpha1/#IstioOperatorSpec)
 for the complete set of configuration settings.
 
+## Canary Upgrade
+
+You can use the standalone operator to do canary upgrade of Istio control plane, the process is similar to the [canary upgrade with `istioctl`]((/docs/setup/upgrade/#canary-upgrades))
+
+For example, if you used the operator to install Istio in the previous section, verify that the `example-istiocontrolplane` custom resource for the `IstioOperator` exists in your cluster::
+
+{{< text bash >}}
+$ kubectl get iop --all-namespaces
+NAMESPACE      NAME                        REVISION   STATUS    AGE
+istio-system   example-istiocontrolplane              HEALTHY   11m
+{{< /text >}}
+
+Then run the following command to install the new revision of the Istio control plane based on the in-cluster `IstioOperator` Custom Resource:
+
+{{< text bash >}}
+$ istioctl operator init --revision 1-7-0
+{{< /text >}}
+
+{{< tip >}}
+You can alternatively use Helm to deploy the operator of a different by setting revision value:
+
+{{< text bash >}}
+$ helm template manifests/charts/istio-operator/ \
+  --set hub=docker.io/istio \
+  --set tag={{< istio_full_version >}} \
+  --set operatorNamespace=istio-operator \
+  --set revision=canary \
+  --set watchedNamespaces=istio-system | kubectl apply -f -
+{{< /text >}}
+
+Note that you need to [download the Istio release](/docs/setup/getting-started/#download)
+to run the above command.
+{{< /tip >}}
+
+After running the command, you will have two control plane deployments and services running side-by-side:
+
+{{< text bash >}}
+$ kubectl get pods -n istio-system -l app=istiod
+NAME                             READY   STATUS    RESTARTS   AGE
+istiod-5f4f9dd5fc-4xc8p          1/1     Running   0          10m
+istiod-1-7-0-55887f699c-t8bh8    1/1     Running   0          8m13s
+{{< /text >}}
+
+{{< text bash >}}
+$ kubectl -n istio-system get svc -l app=istiod
+NAME            TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)                                         AGE
+istiod          ClusterIP   10.87.7.69   <none>        15010/TCP,15012/TCP,443/TCP,15014/TCP,853/TCP   10m
+istiod-canary   ClusterIP   10.87.4.92   <none>        15010/TCP,15012/TCP,443/TCP,15014/TCP,853/TCP   7m55s
+{{< /text >}}
+
 ## Uninstall
 
-Delete the Istio deployment:
+Delete the Istio deployment. Note that if you have multiple operators with different revisions running, this might result in uninstalling Istio of different revisions. 
```

Comment:
```suggestion
If you used the operator to perform a canary upgrade of the control plane, you can uninstall the old control plane and keep the new one by running the following command:

{{< text bash >}}
$ istioctl operator remove --revision <revision>
{{< /text >}}

Otherwise, delete the in-cluster `IstioOperator` CR, which will uninstall all revisions of Istio that may be running:
```

Reply:
done